api/pokemon.tsx

```diff
@@ -4,9 +4,12 @@ interface Pokemon {
   height: number;
   base_experience: number;
   sprites: [object];
+  url: string;
 }
 
-export default async function getFavPokemon(favPokemonName) {
+export default async function getFavPokemon(
+  favPokemonName: string
+): Promise<Pokemon> {
```

Comment:
I think it should be `Promise<Pokemon | null>` -> is TypeScript happy with the current approach?